### PR TITLE
November release extension updates

### PR DESCRIPTION
**Pull request by llali**
Please include the November release extension updates in this PR, and this will be used to update the RC build gallery during the test signoff, and will be merged and published on release day.

### CODE REVIEW

**Review by kburtram — APPROVED** [review]

**Comment by alanrenmsft** [comment]
@llali can the insider feed update wait until we publish the Nov release? if you want to the insider release go out earlier, please open a new PR to only update the insider feed. I am going to use your PR as the master 'November release extension updates'

**Comment by llali** [comment]
> @llali can the insider feed update wait until we publish the Nov release? if you want to the insider release go out earlier, please open a new PR to only update the insider feed. I am going to use your PR as the master 'November release extension updates'

@alanrenmsft  sure will create seperate PR for insider

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -3090,14 +3090,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.4.0",
-							"lastUpdated": "10/06/2020",
+							"version": "0.5.0",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:3093]: Generally it's useful to have insiders released immediately so we can have that channel be testing it as well. Is there any specific reason you'd want to wait until the release happens to update insiders?

Comment (Charles-Gagnon) [reply extensionsGallery-insider.json]: oops, didn't see the other comments. Disregard =)

**Comment by alanrenmsft** [comment]
@abist is your vsix for agent ready? and tested?

**Comment by abist** [comment]
@alanrenmsft It's in port request - https://github.com/microsoft/azuredatastudio/pull/13232

**Comment by alanrenmsft** [comment]
@abist ,I see, please only update the PR when the vsix is ready and tested properly, we may copy content from here to RC directly anytime. it is ok this time, because the ML extension is also not ready yet.

**Comment by alanrenmsft** [comment]
I have taken care of the commits here following the new process and will close this PR now.

**Pull request closed without merging**